packages/components/src/ResourceList/ResourceList.scss

```diff
@@ -1,13 +1,19 @@
-$tc-resource-list-height: 25rem !default;
-
+$tc-resource-list-margin-top: 2rem !default;
+$tc-resource-list-height: calc(25rem - #{$tc-resource-list-margin-top}) !default;
 .tc-resource-list {
 	:global(.tc-list-list) {
+		padding: 0;
 		padding-left: 0;
 		padding-right: 0;
 	}
 
+	:global(.tc-resource-list-toolbar) form {
+		margin-top: $padding-small;
+	}
+
 	&-items {
 		height: $tc-resource-list-height;
+		margin-top: $tc-resource-list-margin-top;
```

Comment:
It will impact ResourcePicker too
![image](https://user-images.githubusercontent.com/18534166/75786544-c36cf800-5d65-11ea-8626-3a95f8876873.png)

Reply:
done